src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -528,6 +528,11 @@
   - name: Create a scope map that allows content/write and metadata/read actions for `hello-world` repository, and content/read action for `hello-world-again`.
     text: >
         az acr scope-map create -n MyScopeMap -r MyRegistry --repository hello-world content/write metadata/read --repository hello-world-again content/read --description "Sample scope map."
+  - name: Create a scope map that allows content/write and metadata/read actions for `hello-world` repository, and all gateway actions for `connectedRegistry`.
```

Comment:
I would expect most cases need content/read to do image pull. So I would suggest the sample includes content/read and metedata/write

Reply:
I'll change the example to add all the actions.